modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadThrowLaws.kt

```diff
@@ -0,0 +1,57 @@
+package arrow.test.laws
+
+import arrow.Kind
+import arrow.core.extensions.eq
+import arrow.test.generators.GenK
+import arrow.test.generators.fatalThrowable
+import arrow.test.generators.throwable
+import arrow.typeclasses.Apply
+import arrow.typeclasses.Eq
+import arrow.typeclasses.EqK
+import arrow.typeclasses.Functor
+import arrow.typeclasses.MonadThrow
+import arrow.typeclasses.Selective
+import io.kotlintest.fail
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import io.kotlintest.shouldThrowAny
+
+object MonadThrowLaws {
```

Comment:
I believe you should probably hook this up in `MonadDeferLaws`'s laws functions as:

```
BracketLaws.laws(...) +
  MonadThrowLaws.laws(...) +
      monadDeferLaws(...)
``` 

Although I'm unsure if this won't run all MonadError tests twice as both Bracket and MonadThrow come from MonadError, but independently

Reply:
yeah, I guess that should be aligned so it doesn't execute the tests more times than required

Reply:
Ah nevermind, I just noticed that in `testLaws` function there's a distinct! So good to go with the suggested change

Reply:
perfect then